public/index.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
```

Comment:
I don't see where this was before. Or was it implicitely given by neutrino?

Reply:
I added it